### ISSUE TITLE
[Suggestion] Crafting Recipe, Paper Cartridge Texture and Third Person reload animation:

### ISSUE THREAD

**Issue by DegeneracyIsNotOK** [ewewukek/mc-musketmod#5]
Firstly I would like to express my "love" for the mod. It's simple and it sticks to the mechanics and art style of the game. 
Because of these features it can find it's way into any vanilla styled mod-pack.

However there are a few things that **personally** bug me and improvements that **I feel** could be made.

### First of these things that I would like to bring up is the crafting recipe:
You are forced create two components, the Gun Barrel and the Gunstock, in order to craft yourself a Musket. These two components however are completely useless for anything else excluding the aforementioned Musket crafting recipe.
I would like to suggest changing the crafting recipe so that it would instead of requiring these useless components, instead use vanilla items, which would in turn further simplify the mod.
![Screenshot_14](https://user-images.githubusercontent.com/73244061/113611380-b8faed00-964e-11eb-87c7-201314d63588.png)
_(Pictured above is my personal suggestion for the crafting recipe requiring only vanilla items)_

### Second thing that i would want to bring up is the fact that the paper cartridge has rims:
With rimmed ammunition being a staple of modern ammunition that uses metal casings, I strongly believe that the texture of the paper cartridge should be tweaked. Additionally to further fit the vanilla art style and to look less **chonky** in comparison to the musket itself the texture could be tweaked to be slightly angled to the right in the same fashion that arrows are.
![Screenshot_15](https://user-images.githubusercontent.com/73244061/113616148-08441c00-9655-11eb-992a-ff79d8643735.png)
_(There are many variations of paper ammunition, my take, pictured above, is one that has the projectile covered as well)_

### The very last thing that I would want to bring up are the third person reload animations, or the lack of.
It might be just my game but when I reload the musket in third person there isn't an animation of any kind. This is quite glaring, especially in multiplayer. Adding a third person animation that matches the reload animation if first person would vastly improve the quality of the mod.

_(Also as a side note please do capitalize the M in mod like a normal human being)_
![Screenshot_10](https://user-images.githubusercontent.com/73244061/113619390-31ff4200-9659-11eb-884f-ec059d849bb9.png)

**Comment by ewewukek** [comment]
thanks for feedback!

1. barrel and stock are purely decorative. I've added them because other mods had them. setting their recipe result to "minecraft:air" disables them.
2. I agree it does look like a rim. it supposed to resemble a twisted paper end, but I'm a bad artist) rotating it 45° is a simple idea I have not think of, I'll try that. thanks!
3. I have plans changing current ugly implementation of first-person animation, I'll add third-person animations to that plans)

(mod title on curseforge page is fixed, in-game title is also capitalized)

**Comment by DegeneracyIsNotOK** [comment]
No, thank **you** for taking my suggestion into your consideration!
I'll just provide _one_ more comprehensive reasoning for the crafting change suggestion I failed to explain originally.
Not a single item in the game requires a specialized component that has no other uses besides a single crafting recipe. An example I'm going to provide is definitely the crossbow. Despite it's unique and complex shape it doesn't use any specialized components such as "Crossbow Body", rather it opts out to use normal vanilla items that are used in many other crafting recipes, such as sticks and string. And by using the tripwire hook it gives an additional use to an item that can be used for other things. The two components the mod adds are basically useless besides the Musket crafting recipe, straying away from **this** vanilla mentality where everything has a use of some kind. Additionally with the whole crossed out image thing I meant to say that with the removal of these two components the mod would be even more lightweight that it already is.

If I have failed to _**persuade**_ you with this **overly** long explanation, might I suggest allowing the Gunstock to be used as a fuel source and allowing the Gun Barrel to be smelted back _(recycled)_ into an iron nugget. Which would in turn allow you to use these two components for something other than the crafting recipe if you accidentally crafted them, for an example.
![Furnace](https://user-images.githubusercontent.com/73244061/113945941-aaa40100-9807-11eb-890d-1552b67f286c.png)
Either way, sincerely thank you for implementing one of my suggestions so quickly, and the magnificent mod you are working on.
